dist/index.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+var core = require("./core"),
+    modules =
```

Comment:
...Not sure what is happing here?  The line break makes it confusing if you're trying to store angular.module("c2fo.ui",...)